index.html

```diff
@@ -168,33 +183,27 @@ <h3>MediaDevices.produceCropTarget</h3>
           </dt>
           <dd>
             <p>
-              The first call of {{MediaDevices/produceCropTarget}} on an {{HTMLElement}} of a
-              supported type, permanently associates that element with a {{CropTarget}}. Subsequent
-              calls return the same {{CropTarget}}. This {{CropTarget}} may be used as input to
```

Comment:
(lost this comment somehow)

"Treated as equal" is an optimization that should be taken based on underlying realities in the cropTo algorithm, orthogonally to any API decision here. Instead, what seems at stake here is merely:
```js
const ct_a1 = produceCropTarget(document.getElementById('id_a'));
const ct_a2 = produceCropTarget(document.getElementById('id_a'));
console.log(ct_1 === ct_2); // true or false?  
```
I do agree though this can be left for a separate issue.

Reply:
We no longer need to specify these rules as they are connected to the same element.
By having a weak reference slot, this kind of ambiguities disappear.